CMakeLists.txt

```
@@ -210,15 +210,18 @@ IF(CPUINFO_SUPPORTED_PLATFORM)
       LIST(APPEND CPUINFO_SRCS
         src/arm/android/properties.c)
     ENDIF()
-  ELSEIF(CPUINFO_TARGET_PROCESSOR MATCHES "^(riscv(32|64))$")
+  ELSEIF (CPUINFO_TARGET_PROCESSOR MATCHES "^(riscv(32|64)?)$")
     LIST(APPEND CPUINFO_SRCS
+      src/riscv/cache.c
       src/riscv/uarch.c)
     IF(CMAKE_SYSTEM_NAME STREQUAL "Linux")
       LIST(APPEND CPUINFO_SRCS
+        src/riscv/linux/cpuinfo.c
+        src/riscv/linux/hwcap.c
         src/riscv/linux/init.c
-	src/riscv/linux/riscv-hw.c
-	src/riscv/linux/riscv-isa.c)
-    ENDIF()
+        src/riscv/linux/riscv-hw.c
+        src/riscv/linux/riscv-isa.c)
+    ENDIF ()
```

Why are you adding white-ace here?
```suggestion
    ENDIF()
```

---

Removed whitespace